extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1768,6 +1768,17 @@ export class AmpA4A extends AMP.BaseElement {
   isVerifiedAmpCreative() {
     return this.isVerifiedAmpCreative_;
   }
+
+  /**
+   * Returns a promise that will resolve when the friendly iframe's window's
+   * `onload` event has been emitted.
```

Comment:
Apologies, I'm changing my mind on this as I want to make sure this function is only called at the appropriate point in the lifecycle.  Can you instead modify onCreativeRender to provide an additional parameter which is an onLoadPromise.  You can decide if you want that to be populated if FIE and non-FIE (just listen for load on xdomain frame).

Reply:
Done.